LogMonitor/src/LogMonitor/LogFileMonitor.cpp

```diff
@@ -1696,12 +1698,20 @@ void LogFileMonitor::WriteToConsole( _In_ std::wstring Message, _In_ std::wstrin
         if (msg.size() > 0) {
             // escape backslashes in FileName
             auto fmtFileName = Utility::ReplaceAll(FileName, L"\\", L"\\\\");
-            // sanitize msg
-            Utility::SanitizeJson(msg);
-            auto log = Utility::FormatString(logFmt, msg.c_str(), fmtFileName.c_str());
-            logWriter.WriteConsoleLog(log);
-        }
+            std::wstring logFmt;
+            if (Utility::CompareWStrings(m_logFormat, L"JSON"))
+            {
+                logFmt = L"{\"Source\": \"File\",\"LogEntry\": {\"Logline\": \"%s\",\"FileName\": \"%s\"},\"SchemaVersion\":\"1.0.0\"}";
+                // sanitize message
+                Utility::SanitizeJson(msg);
+            } else {
+                logFmt = L"<Log><Source>File</Source><LogEntry><Logline>%s</Logline><FileName>%s</FileName></LogEntry></Log>";

```

Comment:
nit: same here, this can be combined with L1701 and you do away with the `else` part.